src/api.js

```diff
@@ -1,25 +1,27 @@
 "use strict";
 
-var express       = require('express');
-var Debug         = require('debug');
-var Promise       = require('promise');
-var uuid          = require('uuid');
-var hawk          = require('hawk');
-var aws           = require('aws-sdk');
-var assert        = require('assert');
-var _             = require('lodash');
-var bodyParser    = require('body-parser');
-var path          = require('path');
-var fs            = require('fs');
-require('superagent-hawk')(require('superagent'));
-var request       = require('superagent-promise');
-var scopes        = require('taskcluster-lib-scopes');
-var crypto        = require('crypto');
-var cryptiles     = require('cryptiles');
-var taskcluster   = require('taskcluster-client');
-var Ajv           = require('ajv');
-var errors        = require('./errors');
-var typeis        = require('type-is');
+import * as express from 'express';
+import * as _ from 'lodash';
+import * as scopes from 'taskcluster-lib-scopes';
+import * as errors from  './errors';
+import Debug from 'debug';
+import Promise from 'promise';
+import uuid from 'uuid';
+import hawk from 'hawk';
+import aws from 'aws-sdk';
+import assert from 'assert';
+import bodyParser from 'body-parser';
+import path from 'path';
+import fs from 'fs';
+import request from 'superagent-promise';
+import crypto from 'crypto';
+import cryptiles from 'cryptiles';
+import taskcluster from 'taskcluster-client';
+import Ajv from 'ajv';
+import typeis from 'type-is';
+import 'superagent-hawk';
+import 'superagent';
+  
```

Comment:
you are missing `require('superagent-hawk')(require('superagent'));`

Reply:
you have the imports... but not the expression that does the injection...

Reply:
Yeah. I removed the imports and added `require('superagent-hawk')(require('superagent'));`  in the next commit . Thanks

Reply:
you can still do it with import if you really want... there is two imports and one function invocation in that expression...

IMO; leaving it as require is fine :)